src/modules/Crowbar.rb

```diff
@@ -272,11 +287,22 @@ def Write
         Report.Error(Message.ErrorWritingFile(@network_file))
       end
 
-      @template_crowbar["attributes"]["crowbar"]["users"]       = @users
       unless hash2json(@template_crowbar, @crowbar_file)
         Report.Error(Message.ErrorWritingFile(@crowbar_file))
       end
 
+      begin
+        crowbarrc = IniFile.load("/etc/crowbarrc")
+        if crowbarrc["default"]["username"] != @admin_user ||
```

Comment:
This might crash when `crowbarrc["default"]` is missing (is `nil`).

Reply:
See https://github.com/yast/yast-crowbar/pull/63#discussion_r273005695